docs/install.md

```diff
@@ -5,78 +5,99 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Spark is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+Spark is available in the Universe and can be installed by using either the GUI or the DC/OS CLI.
 
-##  <a name="install-enterprise"></a>Prerequisites
```

Comment:
What do you mean?  Why is it not being a referenced a reason to add it?  Shouldn't we only add it if it *is* referenced?